services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -352,32 +354,45 @@ async def _run_trigger_observation_queue_task(self) -> None:
 
         async def _observing_single_service(service_name: str) -> None:
             scheduler_data: SchedulerData = self._to_observe[service_name]
+            dynamic_sidecar_settings: DynamicSidecarSettings = (
+                self.app.state.settings.DYNAMIC_SERVICES.DYNAMIC_SIDECAR
+            )
 
-            # NOTE: ANE: not very readable should be refactored
-            if (
+            observation_cycle_raised_error = (
                 scheduler_data.dynamic_sidecar.status.current
                 == DynamicSidecarStatus.FAILING
-            ):
-                # Nothing will be done if there is an error while interacting
-                # with the sidecar.
+            )
+            if observation_cycle_raised_error:
+                # `Observation cycle is skipped` after an error while
+                # interacting with the sidecar.
                 # It makes no sense to continuously occupy resources or create
-                # issues due to high request to components like the `docker damon`
+                # issues due to high request to components like the `docker daemon`
                 # and the `storage service`.
 
-                # TODO:
-
-                # MIGHT be a better alternative?
-                # - if node was deleted! (you know because in theory you call
-                # remove_node without saving ths state) just cleanup sidecars
-                # and remove from observation cycle [allows users to delete nodes and
-                # still release resources]
-
-                # FOR sure we want this
-                # - if the sidecar is missing (remove the proxy if it exists)
-                # and then remove from observation cycle [allows user to manually
-                # remove failing sidecars and the entire thing to still work afterwards]
+                failed_while_saving_state_and_outputs = (
+                    scheduler_data.dynamic_sidecar.service_removal_state.can_save
+                    and scheduler_data.dynamic_sidecar.were_services_created
+                )
+                if failed_while_saving_state_and_outputs:
+                    # Since user data is important and must be saved, take no further
+                    # action and wait for manual intervention from support.
```

Comment:
@GitHK see the proposal I did for the same issue below. the graylog alert is a very good idea. @mrnicegyu11 can you define one, or that would be a ENV variable ?

Reply:
I feel like the POs/user expect us to act immediately at this point if a save fails. So IF and HOW we add an alert can be configured (we can silence for a period of time after it triggers), but we should log a very clear and unique (thus queryable) error message here in case this happens (read: loglevel error). 

Then, afterwards, we can still discuss how, when, and for how long we actually alert when this happens.

Question: Why would it be triggered every 5 seconds? if the saving fails, it would be sufficient to log once.

Sorry for being insistent here. What is the other scenario? We dont get alerted and it is up to batmans to check every container individually whether it is hanging or properly up daily?